wremnants/datasets/datagroups2016.py

```diff
@@ -93,7 +93,9 @@ def make_datagroups_2016(input_file, combine=False, pseudodata_pdfset = None, ap
             color = "grey",
             selectOp = fakeOp,
         )
-
+        dg.filterGroups(filterGroups)
+        dg.excludeGroups(excludeGroups)
```

Comment:
Just move the filter call to after the if dg.wmass, outside the if statement (vs. calling it twice) ?

Reply:
It wouldn't work, because then the fakes would not be made only with the selected processes. Not that fakes would make sense either when excluding processes, but the logic should stay consistent, in that fakes plus selected processes should sum up to data in the control regions

Reply:
I don't understand why the filter needs to be called twice

Reply:
If you don't call the filters before the fakes are created, then the fakes will be created using all previous groups, including those that you would have excluded, so this is inconsistent.
And then, if you don't apply the filter a second time after the fakes, the fakes will never be allowed to be excluded. In the past this could not happen because the groups were existing in a different location, and when the filter was applied all groups were already defined